packages/pyright-internal/src/languageServerBase.ts

```diff
@@ -505,6 +507,7 @@ export abstract class LanguageServerBase implements LanguageServerInterface, Dis
         this.connection.onDidCloseTextDocument(async (params) => this.onDidCloseTextDocument(params));
         this.connection.onDidChangeWatchedFiles((params) => this.onDidChangeWatchedFiles(params));
         this.connection.workspace.onWillRenameFiles(this.onRenameFiles);
```

Comment:
when a file gets renamed, rename it in the baseline file

Reply:
![image](https://github.com/user-attachments/assets/d7589638-a564-4bf6-a0c2-1f1106b9bc87)

thanks github for hiding this unresolved thread underneath this, ffs